enterprise/frontend/src/embedding-sdk/lib/interactive-question/run-question-on-load.ts

```diff
@@ -35,8 +40,26 @@ export const runQuestionOnLoadSdk =
     const originalQuestion =
       originalCard && new Question(originalCard, metadata);
 
+    let question = new Question(card, metadata);
+
+    question = syncCardParametersWithTemplateTags(question);
+
+    const queryParams = initialSqlParameters
+      ? mapObject(initialSqlParameters, String)
```

Comment:
The SQL parameter values are passed as strings, as we already denoted the type in the native query editor.